### DISCUSSION TITLE
how to override boostrap using css

### DISCUSSION THREAD

**Discussion by cemmanzy** [twbs/bootstrap#37254] [Q&A]
i tried customizing my boostrap using css but it not working

**Comment by nagualdev** [comment]
you need to put the CSS variables inside a CSS selector.
Example:

.nav-link {
    --bs-nav-link-hover-color: #06525D ;
    --bs-nav-link-color: green;
    --bs-nav-link-disabled-color: white;
    --bs-nav-link-font-weight: bold;
}
